modules/installation-azure-user-infra-uploading-rhcos.adoc

```diff
@@ -57,8 +60,13 @@ that matches your {product-title} version if it is available.
 
 . Copy the chosen VHD to a blob:
 +
+[source,terminal]
```

Comment:
@e-tienne, what's the difference between these two commands?

Reply:
Hello @kalexand-rh , the first one creates a container (analogous to a local folder on your computer) and the second copies the "rhcos.vhd" blob to "vhd" container. You can see a breakdown of typical steps used with blobs here: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli